Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.yml

```diff
@@ -992,6 +992,13 @@ script:
     - name: attachCIDs
       description: A comma-separated list of CIDs to embed attachments within the email itself.
       isArray: true
+    - description: Whether to handle inline images in the HTML body. When set to 'True', inline images will be extracted from the HTML body and will be attached to the email as an inline attachment object.
```

Comment:
- description: Whether to handle inline images in the HTML body. When set to 'True', inline images are extracted from the HTML body and attached to the email as an inline attachment object.